packages/debs/SPECS/debian/postrm

```diff
@@ -24,6 +27,14 @@ case "$1" in
         if [ -d "/run/wazuh-server" ]; then
             rm -rf /run/wazuh-server || true
         fi
+
+        if [ -d "/etc/wazuh-server" ]; then
+            rm -rf /etc/wazuh-server || true
+        fi
+
+        if [ -d "/var/log/wazuh-server" ]; then
+            rm -rf /var/log/wazuh-server || true
+        fi
```

Comment:
What about the folder `/usr/share/wazuh-server`? Shouldn't we remove it?

Reply:
As we can see in the `find` post purge the `/usr/share/wazuh-server` is deleted.